x-pack/plugins/apm/public/components/app/service_map/popover/anomaly_detection.tsx

```diff
@@ -78,7 +79,10 @@ export function AnomalyDetection({ serviceName, serviceAnomalyStats }: Props) {
           <h3>{ANOMALY_DETECTION_TITLE}</h3>
         </HealthStatusTitle>
         &nbsp;
-        <EuiIconTip type="iInCircle" content={ANOMALY_DETECTION_TOOLTIP} />
+        <EuiToolTip position="top" content={ANOMALY_DETECTION_TOOLTIP}>
```

Comment:
Is this a bug on `EuiIconTip`? Also you can delete the comment below

Reply:
Thanks I forgot to delete it.
I don't know if it's a bug, not sure how is the expected behaviour of `EuiIconTip`

Reply:
This seems like it might be a bug in EUI, EuiIconTip should work here.

Reply:
@ogupte I will ask them about this issue. 
Do you think in this case we need to use the `EuiIconTip` or we can use the `EuiTooltip`?

Reply:
hi @ogupte @gbamparop @formgeist, I put back the `EuiIconTip`, and set the `initialFocus` on the popover to false, so the focus gets to the popover itself, I follow the suggestion from the EUI team in order to keep also the accessibility of the component.

Reply:
@MiriamAparicio OK sounds good - the experience will be the same as you've demoed in the PR description? It doesn't show the tooltip upon popover opening?

Reply:
@formgeist yes, will be the same. I updated the recording, it also shows the change of focus using keyboard navigation.

Reply:
@MiriamAparicio Thanks for updating the video, it looks as expected 👍